# Question 1

**a. False**. We already know the sample proportion is $82\%$.

**b. False**. The confidence interval provides an estimate range for the population proportion, not a definitive range for all Americans.

**c. True**. This is a correct interpretation of what a $95\%$ confidence level means.

**d. True**. The margin of error for a proportion is $
\text{ME} = Z \sqrt{\frac{\hat{p}(1 - \hat{p})}{n}}
$
the margin of error is inversely proportional to the square root of the sample size

**e. True**. A majority is more than $50\%$.

# Question 3

## a.

| Treatment     | Virologic Failure | No Virologic Failure | Total |
|---------------|-------------------|----------------------|-------|
| Nevirapine    | $26$              | $94$                 | $120$ |
| Lopinavir     | $10$              | $110$                | $120$ |
| **Total**     | $36$              | $204$                | $240$ |


## b.

- **Null Hypothesis (\( $H_0$ \))**: The virologic failure rates are the same for both treatment groups.
  $$
  H_0: p_{\text{Nevirapine}} = p_{\text{Lopinavir}}
  $$

- **Alternative Hypothesis (\( $H_1$ \))**: The virologic failure rates are different for the two treatment groups.
  $$
  H_1: p_{\text{Nevirapine}} \neq p_{\text{Lopinavir}}
  $$

## c.
$$
\hat{p}_{\text{Nevirapine}} = \frac{26}{120} = 0.2167
$$

$$
\hat{p}_{\text{Lopinavir}} = \frac{10}{120} = 0.0833
$$

$$
p_{pooled} = \frac{26 + 10}{120 + 120} = \frac{36}{240} = 0.15
$$

$$
SE = \sqrt{p_{pooled}(1 - p_{pooled}) \left(\frac{1}{n_{\text{Nevirapine}}} + \frac{1}{n_{\text{Lopinavir}}}\right)} = \sqrt{0.15 \times 0.85 \left(\frac{1}{120} + \frac{1}{120}\right)} = \sqrt{0.15 \times 0.85 \times \frac{2}{120}} = \sqrt{0.002125} \approx 0.0461
$$


$$
z = \frac{\hat{p}_{\text{Nevirapine}} - \hat{p}_{\text{Lopinavir}}}{SE} = \frac{0.2167 - 0.0833}{0.0461} \approx 2.89
$$


$$
p\text{-value} \approx 0.0038
$$


Since the p-value ($0.0038$) is less than the significance level ($\alpha = 0.05 $), we reject the null hypothesis. This means there is sufficient evidence to conclude that the virologic failure rates are different for the two treatment groups.

# Question 4

- **Group 1 (financial help second):**
  - Sample size ($ n_1 $) = $771$
  - Sample proportion ($ \hat{p}_1 $) = $0.47$

- **Group 2 (penalty second):**
  - Sample size $ n_2 $ = $732$
  - Sample proportion ($ \hat{p}_2 $) = $0.34$


$
SE = \sqrt{\frac{0.47 (1 - 0.47)}{771} + \frac{0.34 (1 - 0.34)}{732}} = \sqrt{\frac{0.47 \times 0.53}{771} + \frac{0.34 \times 0.66}{732}} = \sqrt{\frac{0.2491}{771} + \frac{0.2244}{732}} = \sqrt{0.000323 + 0.000307} = \sqrt{0.00063} \approx 0.0251
$


$
ME = 1.645 \times 0.0251 \approx 0.0413
$

$
(\hat{p}_1 - \hat{p}_2) \pm ME = (0.47 - 0.34) \pm 0.0413 = 0.13 \pm 0.0413
CI = (0.0887, 0.1713)
$


We are 90% confident that the difference in approval rates between the two question phrasings lies between $8.87\%$ and $17.13\%$.

# Question 5

| Option   | Observed Frequency (\( O_i \)) |
|----------|-------------------------------|
| Rock     | $43$                          |
| Scissors | $21$                          |
| Paper    | $35$                          |


- **Null Hypothesis (\( $H_0$ \))**: Players choose rock, paper, and scissors with equal probability.
  $$
  H_0: p_{\text{Rock}} = p_{\text{Scissors}} = p_{\text{Paper}} = \frac{1}{3}
  $$

- **Alternative Hypothesis (\( $H_1$ \))**: Players do not choose rock, paper, and scissors with equal probability.
  $$
  H_1: \text{At least one } p_i \neq \frac{1}{3}
  $$


$
O_{\text{Rock}} = 43, \quad O_{\text{Scissors}} = 21, \quad O_{\text{Paper}} = 35
$

Total number of games: \( $43 + 21 + 35 = 99$ \)
$
E_i = \frac{99}{3} = 33 \quad \text{for each option}
$

$df = 2$

$$
\chi^2 = \sum \frac{(O_i - E_i)^2}{E_i}
 = \frac{(43 - 33)^2}{33} + \frac{(21 - 33)^2}{33} + \frac{(35 - 33)^2}{33}
 = \frac{10^2}{33} + \frac{(-12)^2}{33} + \frac{2^2}{33}
 = \frac{100}{33} + \frac{144}{33} + \frac{4}{33}
 = 3.03 + 4.36 + 0.12 = 7.51
$$

$$
$$




- Critical value for \( $df = 2$ \) at \( $\alpha = 0.05$ \): \( $5.99$ \)
- P-value for \( $\chi^2 = 7.51$ \): \( $0.0234$ \)


Since the chi-square test statistic ($7.51$) is greater than the critical value ($5.99$), we reject the null hypothesis at the $0.05$ significance level. This suggests that players do not choose rock, paper, and scissors with equal probability; certain options are favored above others.


# Question 6

| Gender  | Ketchup | Mustard | Relish | Total |
|---------|---------|---------|--------|-------|
| Male    | $15$      | $23$      | $10$     | $48$    |
| Female  | $25$      | $19$      | $8$      | $52$    |
| **Total** | $40$    | $42$      | $18$     | $100$   |


- **Null Hypothesis (\( $H_0$ \))**: There is no relationship between gender and the preferred condiment (they are independent).

- **Alternative Hypothesis (\( $H_1$ \))**: There is a relationship between gender and the preferred condiment (they are not independent).

| Gender  | Ketchup (\( $O_{11}$ \)) | Mustard (\( $O_{12}$ \)) | Relish (\( $O_{13}$ \)) | Total |
|---------|---------|---------|--------|-------|
| Male    | $15$      | $23$      | $10$     | $48$    |
| Female  | $25$      | $19$      | $8$      | $52$    |
| **Total** | $40$    | $42$      | $18$     | $100$   |

   $$
   E_{11} = \frac{48 \times 40}{100} = 19.2
   $$
   $$
   E_{12} = \frac{48 \times 42}{100} = 20.16
   $$
   $$
   E_{13} = \frac{48 \times 18}{100} = 8.64
   $$
   $$
   E_{21} = \frac{52 \times 40}{100} = 20.8
   $$
   $$
   E_{22} = \frac{52 \times 42}{100} = 21.84
   $$
   $$
   E_{23} = \frac{52 \times 18}{100} = 9.36
   $$

   $$
   \chi^2 = \sum \frac{(O_{ij} - E_{ij})^2}{E_{ij}}
   $$

   $$
   =>
   \chi^2 = \frac{(15 - 19.2)^2}{19.2} + \frac{(23 - 20.16)^2}{20.16} + \frac{(10 - 8.64)^2}{8.64} + \frac{(25 - 20.8)^2}{20.8} + \frac{(19 - 21.84)^2}{21.84} + \frac{(8 - 9.36)^2}{9.36}
   $$

   $$
   =>
   \chi^2 = \frac{(-4.2)^2}{19.2} + \frac{2.84^2}{20.16} + \frac{1.36^2}{8.64} + \frac{4.2^2}{20.8} + \frac{(-2.84)^2}{21.84} + \frac{(-1.36)^2}{9.36}
   $$

   $$
   =>
   \chi^2 = \frac{17.64}{19.2} + \frac{8.0656}{20.16} + \frac{1.8496}{8.64} + \frac{17.64}{20.8} + \frac{8.0656}{21.84} + \frac{1.8496}{9.36}
   $$
   
   $$
   =>
   \chi^2 = 0.9188 + 0.3998 + 0.2141 + 0.8481 + 0.3693 + 0.1976 = 2.9477
   $$


   $$
   df = (r - 1)(c - 1) = (2 - 1)(3 - 1) = 1 \times 2 = 2
   $$


- Critical value for \( $df = 2$ \) at \( $\alpha = 0.05$ \) from chi-square table: \( $5.99$ \)
- P-value for \( $\chi^2 = 2.9477$ \) can be found using a chi-square calculator or table.

Since the chi-square test statistic ($2.9477$) is less than the critical value ($5.99$), we fail to reject the null hypothesis at the $0.05$ significance level. This suggests that there is not enough evidence to conclude that there is a relationship between gender and the preferred condiment.


# Question 7

In [ ]:
# Observed frequencies
observed <- c(18, 16, 10, 10, 16)

# Expected frequencies if absences are equally distributed
expected <- rep(sum(observed) / length(observed), length(observed))

# Perform chi-square goodness-of-fit test
chi_square_test <- chisq.test(x = observed, p = expected / sum(expected))

# Print the test results
print(chi_square_test)



	Chi-squared test for given probabilities

data:  observed
X-squared = 4, df = 4, p-value = 0.406



Since the chi-square test statistic ($4$) is less than the critical value ($9.488$), we fail to reject the null hypothesis at the $0.05$ significance level. This suggests that there is not enough evidence to conclude that the days with the highest number of absences do not occur with equal frequencies during the workweek.


# Question 8

In [ ]:
# Load the required library
library(MASS)

# Load the survey data
data("survey")

# Create a contingency table for Smoke and Exer
contingency_table <- table(survey$Smoke, survey$Exer)

# Perform the chi-square test of independence
chi_square_test <- chisq.test(contingency_table)

# Print the contingency table
print(contingency_table)

# Print the test results
print(chi_square_test)


Warning message in chisq.test(contingency_table):
“Chi-squared approximation may be incorrect”


       
        Freq None Some
  Heavy    7    1    3
  Never   87   18   84
  Occas   12    3    4
  Regul    9    1    7

	Pearson's Chi-squared test

data:  contingency_table
X-squared = 5.4885, df = 6, p-value = 0.4828




Since the p-value ($0.4828$) is greater than the significance level ($\alpha = 0.05 $), we fail to reject the null hypothesis. This means there is not enough evidence to conclude that there is a significant association between students' smoking habits and their exercise levels. The warning message indicates that some expected frequencies are low, which can affect the validity of the chi-square approximation. However, based on the provided test results, we proceed with the conclusion from the chi-square test.
